src/wp-includes/css/media-views.css

```diff
@@ -2828,13 +2828,13 @@
 		right: 0;
 		z-index: 9;
 	}
-	
+
 	.media-bg-overlay {
 		content: '';
 		background: #ffffff;
 		width: 100%;
 		height: 100%;
-		display: hidden;
```

Comment:
Looks like it actually needs to be `display: none;` otherwise this breaks the spinner in the media library again.